test/TestThreeHeapOrdering.t.sol

```diff
@@ -222,7 +222,9 @@ contract TestThreeHeapOrdering is TestCommonHeapOrdering {
 
     function testInsertWrap() public {
         MAX_SORTED_USERS = 30;
-        for (uint256 i = 0; i < 30; i++) update(accounts[i], 0, NB_ACCOUNTS - i);
+        for (uint256 i = 0; i < 30; i++) {
+            update(accounts[i], 0, NB_ACCOUNTS - i);
+        }
```

Comment:
That's one thing that I don't like in forge fmt. And afaik it's not configurable. Who's opening an issue / PR for this 🤓